Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,479 @@
+    , "AppRuleObject", sosAppRuleObject
+    , "AppRuleObjectContent", sosAppRuleObjectContent
+    , "AppRuleAction", sosAppRuleAction
+    , "SourceIPv6Address", sosSourceIPv6Address
+    , "DestinationIPv6Address", sosDestinationIPv6Address
+    , "ServiceName", sosServiceName
+    , "AppFullString", sosAppFullString
+    , "AppIDNumber", sosAppIDNumber
+    , "AppID", sosAppID
+    , "AppCategoryID", sosAppCategoryID
+    , "AppSignatureID", sosAppSignatureID
+    , "IPSCategoryName", sosIPSCategoryName
+    , "AntiSpywareCategory", sosAntiSpywareCategory
+    , "URLPathName", sosURLPathName
+    , "FileIdentifier", sosFileIdentifier
+    , "DPIInspectedFlow", sosDPIInspectedFlow
+    , "DestinationPortDPT", sosDestinationPortDPT
+    , "NATDestinationPort", sosNATDestinationPort
+    , "SourcePortSPT", sosSourcePortSPT
+    , "NATSourcePort", sosNATSourcePort
+    , "BladeID", sosBladeID
+    , "AuditID", sosAuditID
+    , "AuditTransactionID", sosAuditTransactionID
+    , "AuditUserPrivileges", sosAuditUserPrivileges
+    , "AuditGroupName", sosAuditGroupName
+    , "AuditGroupIndex", sosAuditGroupIndex
+    , "AuditOldValue", sosAuditOldValue
+    , "AuditNewValue", sosAuditNewValue
+    , "AuditTime", sosAuditTime
+    , "AuditPath", sosAuditPath
+    , "UUID", sosUUID
+    , "HttpRequestMethod", sosHttpRequestMethod
+    , "FileName", sosFileName
+    , "FileSize", FileSize
+    , "HttpReferrer", sosHttpReferrer
+    , "HttpUserAgent", HttpUserAgent
+    , "CaptureATPVerdict", sosCaptureATPVerdict
+    , "Country", sosCountry
+    , "FirewallAction", sosFirewallAction
+    , "ApplicationID", sosApplicationID
+    , "SourceZone", sosSourceZone
+    , "DestinationZone", sosDestinationZone
+    , "CFSCategoryID", sosCFSCategoryID
+    , "CFSCategoryName", sosCFSCategoryName
+    , "CFSPolicyName", sosCFSPolicyName
+    , "AppControlFileName", sosAppControlFileName
+    , "AppCategoryName", sosAppCategoryName
+    , "AppIDName", sosAppIDName
+    , "AppSignatureName", sosAppSignatureName
+    , "IPSFullString", sosIPSFullString
+    , "IPSSignatureName", sosIPSSignatureName
+    , "LegacyMessageCategory", sosLegacyMessageCategory
+    , "LogMsgCategory", sosLogMsgCategory
+    , "LogMsgNote", sosLogMsgNote
+    , "LogMsgSeverity", sosLogMsgSeverity
+    , "NATSourceIPAddress", sosNATSourceIPAddress
+    , "NATDestinationIPAddress", sosNATDestinationIPAddress
+    , "SourceZoneType", sosSourceZone
+    , "DestinationZoneType", sosDestinationZone
+    , "SourceVPNPolicyName", sosSourceVPNPolicyName
+    , "DestinationVPNPolicyName", sosDestinationVPNPolicyName
+    , "InterfaceStatistics", sosInterfaceStatistics
+    , "SoleraNPCSURL", sosSoleraNPCSURL
+    , "SerialNumber", sosSerialNumber
+    , "DeviceModel", sosDeviceModel
+    , "LogMsgID", sosLogMsgID
+    , "RequestURL", RequestURL
+    , "Activity", EventMessage
+    , "EventMessageDetail", sosEventMessageDetail
+    , "AdditionalExtensions", AdditionalExtensions

```

Comment:
@jaimeesc we are extracting required info from AdditionalExtension already in the initial steps. Do we still need to map this additionalfields? Wouldn't this be duplicate info?